queries/slint/highlights.scm

```diff
@@ -18,19 +18,29 @@
 (value) @number
 (string) @string
 
+"for" @repeat
+
+"@" @keyword
+
+[
+"import" 
+"from"
+] @include
+
+[
+"if"
+"else"
+] @conditional
+
+
 [
 "struct"
 "property"
 "callback"
-"import"
-"from"
 "root"
 "parent"
 "this"
```

Comment:
Is `this` a variable.builtin/constant.builtin? I don't know what the semantics of `in` is
 In some languages it is a keyword.operator

Reply:
Oh you're right, they should be variable.builtin. And after looking at other languages queries it seems like `in` should be `@repeat`